### ISSUE TITLE
3.0 - "Chip" Does Not Have Sufficient Contrast

### ISSUE THREAD

**Issue by spencerfinnell** [easydigitaldownloads/easy-digital-downloads#7731]
### Expected behavior

When using the "Chip" component (`<span class="edd-chip"></span>`) the rendered chip should have sufficient contrast between the text and the background color.

### Actual behavior

There is not enough contrast.

### Steps to reproduce the behavior
1) View "Downloads > Customers > Edit Customer > Customer Emails"

<img width="293" alt="Screen Shot 2020-04-13 at 11 57 09 AM" src="https://user-images.githubusercontent.com/491124/79135803-ed123980-7d7d-11ea-8407-1b9d3365ce1f.png">

### Information (if a specific version is affected):
EDD Version (or branch): `release/3.0`

Any other relevant information:

Originally reported in https://github.com/easydigitaldownloads/easy-digital-downloads/issues/7725#issue-597387841 with follow up via https://github.com/easydigitaldownloads/easy-digital-downloads/issues/7725#issuecomment-611623947

**Comment by LisaCee** [comment]
PR #7735
